telemetry/specs/INT.mdk

```diff
@@ -232,6 +258,84 @@ a consistent way to ensure interoperability between the devices.
 Operators of an INT domain should deploy INT Sink capability at domain edges to
 prevent INT information from leaking out of the domain.
 
+# INT Modes of Operation
+Since INT was first introduced at P4.org in 2015, a number of variations of INT
+have been evolved and discussed in IETF and industry communities.
+Also the term 'INT' has been used to broadly indicate data plane
+telemetry in general, not limited to the original classic INT where both instructions
+and metadata are embedeed in the data packets. Hence we define different modes of
+INT operations based on the following criteria:
+
+* whether INT is applied to real data packets vs synthetic (clone or probe) packets.
+* the degree of packet modifications, i.e., what to embed in the packets.
+
+Figure [fig-int-modes] summarizes the different modes of operation and we detail
+them in the following texts.
+
+## INT Applied to Real Data Packets
+
+In this category, the original data packets are monitored and may be modified
+to carry INT instructions and metadata.
+There are three variations based on the level of packet modifications.
+
+* **INT-XD** (eXport Data): network devices directly export metadata from
+their dataplane to the monitoring system based on the INT instructions configured
+at their Flow Watchlists. No packet Modification is needed. This mode was also
```

Comment:
Good point. 
Since we're embracing postcard also under the INT umbrella, I think it's good to give such context so that readers can better connect the dots. Postcard is a quite well known concept/term due to the NSDI'14 paper.
I changed the text to 
"... known as "Postcard" mode in the previous versions of Telemetry Report spec."
also added a reference to the NSDI paper.